### DISCUSSION TITLE
Stuttering audio with use_precision_timing = "auto"

### DISCUSSION THREAD

**Discussion by jgneff** [mikebrady/shairport-sync#1462] [Q&A] [answered]
I just upgraded to Raspberry Pi OS 11 (bullseye) from Raspberry Pi OS 10 (buster), which upgraded Shairport Sync to version 3.3.7 from version 3.2.2. I'm on a Raspberry Pi 3 Model A Plus Rev 1.0 with an IQaudio DAC Pro hooked up to a Tivoli Audio system.

The older Raspberry Pi OS and Shairport Sync versions worked perfectly with my Mac mini and Tivoli Audio, but the newer versions cause severe stuttering in the audio playback on the Tivoli Audio. I can get it to work perfectly again by setting `use_precision_timing = "no"` as shown below in `/etc/shairport-sync.conf`:

```
alsa = 
{
        output_device = "hw:IQaudIODAC";
        mixer_control_name = "Digital";
        use_precision_timing = "no";
}
```

Was this likely fixed by "Fix a bug in the alsa precision timing code" in [Shairport Sync 3.3.8][1]? Am I missing much by disabling the precision timing? The local Mac mini speakers and the Tivoli Audio speakers sound in sync to my ears even with the "standard" timing.

Thank you!
John

[1]: https://github.com/mikebrady/shairport-sync/blob/master/RELEASENOTES.md#version-338

**Comment by mikebrady** [comment, marked as answer]
Thanks. You're missing nothing by turning the precision timing off. But it might be worth updating whenever you get a chance.
